modules/engage-ui/src/main/resources/ui/js/app/engage-ui.js

```diff
@@ -675,19 +675,15 @@ function($, bootbox, _, alertify) {
 
                     if (data.mediapackage.media && data.mediapackage.media.track) {
                     	// Check if there's a 'live' track
-                        for (var i = 0; i < data.mediapackage.media.track.length; i++) {
-                            var track = data.mediapackage.media.track[i];
-                            if (track.live) {
-                            	// Is event in progress?
-                            	var start = new Date(data.mediapackage.start);
-                            	var end = new Date(start.getTime() + parseInt (data.mediapackage.duration));
-                            	var now = new Date(); 
-                                if (now < start || now > end) {
-                                	live = msg_live_not_in_progress;
-                                	canLaunch = false;
-                                } else live = msg_live_in_progress;
-                                break;
-                            }
+                        if (data.mediapackage.media.track.live) {
+                            // Is event in progress?
+                            var start = new Date(data.mediapackage.start);
+                            var end = new Date(start.getTime() + parseInt (data.mediapackage.duration));
+                            var now = new Date();
+                            if (now < start || now > end) {
+                               	live = msg_live_not_in_progress;
+                               	canLaunch = false;
+                            } else live = msg_live_in_progress;
                         }
                     }
                     tile = tile + "<div class=\"live\">" + live + "</div>";
```

Comment:
`… + _.escape(live) + …` would be safer

Reply:
`_.escape(live)` leaves a <br> in the not live videos, it is better to put in the messages before that line.